metadata/definition/mock.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
is it just used for test case?

Reply:
yes

Reply:
I do not think it is a problem. Dubbo Java has many similar practices.

Reply:
If just for test ,then i  think that is a problem, because it can be use out of test case .

what do you think ? @flycash @fangyincheng @AlexStocks